src/sentry/integrations/vsts/vsts_subscription_script.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import
+
+import logging
+
+from django.core.urlresolvers import reverse
+from uuid import uuid4
+
+from sentry.constants import ObjectStatus
+from sentry.integrations.exceptions import ApiError, ApiUnauthorized
+from sentry.models import OrganizationIntegration
+from sentry.utils.http import absolute_uri
+
+VSTS_WEBHOOK_URL = absolute_uri(reverse('sentry-extensions-vsts-issue-updated'))
```

Comment:
This variable seems unused. I'd also say we shouldn't do this as module import time, and if you needed to use it, just calculate it when needed.

Reply:
Ahhhhh I know why I didn't. It was used in a previous draft of this code then, I forgot to remove it.